### PR TITLE
Adds new lobby music - Robocop harmonica, by nuklearcellphoneg

### PR DESCRIPTION
**Pull request by MadmanMartian**
:cl:
 * rscadd: Adds some new lobby music, by nuklearcellphoneg

### CODE REVIEW

File: code/game/gamemodes/gameticker.dm [hunk 1]
```diff
@@ -60,6 +60,7 @@ var/datum/controller/gameticker/ticker
 		"sound/music/starman.ogg",
 		"sound/music/dawsonschristian.ogg",
 		"sound/music/carmenmirandasghost.ogg",
+		"sound/music/robocobharmonica.ogg",
```
Comment (MadmanMartian) [inline code/game/gamemodes/gameticker.dm]: whoops!

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
Very post-ironic memery, I hate it

**Comment by nuklearcellphoneg** [comment]
Pls tell me what 2 do 2 make it better

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
I can't really comment on the quality of the song itself because I'm not really interested in harmonica but as a lobby song, it just doesn't really come out as spess-related
I also can't hear the robocop no matter how hard I try

**Comment by Pathid** [comment]
Would be more fitting as deep space ambiance. Wasn't that the initial idea?

**Comment by beesil** [comment]
Harmonica fits nicely for mining ambience.

**Comment by nuklearcellphoneg** [comment]
It's played much more slowly and it's played on a swing beat. That means the notes are normally played at  1 2 3 4, but in this they're played more 1 23 4. The backing chords are also played by a picked banjo instead of just repeating each chord rhythmically ten times.

https://clyp.it/tu3riry3

That's it, but sped up. You should be able to recognize the riff.

If you still can't, here it is played without swing beat with normal chords instead of banjo picking.

https://clyp.it/ldgvbyeb

And honestly the song itself could be improved, because it's missing the call and answer section from later in the song as well as the downward arpeggio marking the bridge. I wouldn't mind working on it longer and see where it goes from there.

**Comment by Riekland** [comment]
that's cool and all but it doesn't make the track this is adding sound like robocop

**Comment by nuklearcellphoneg** [comment]
Then I'll work on it more if you can't hear it- I mean like I said, it is the song, just slowed down a lot and in a swing beat.

**Comment by nuklearcellphoneg** [comment]
https://clyp.it/phtqhlva

It's about finished, could use some polish.

**Comment by nuklearcellphoneg** [comment]
Thanks

**Comment by nuklearcellphoneg** [comment]
https://soundcloud.com/aintaintcountry/robocop-but-its-harmonica

This is what I've made so far. It kind of drags on in certain spots- any sort of feedback would be appreciated, I can add some flair to some bits and rearrange it some more and re-record the backing track, but this is pretty spot on as far as the main melody goes.

**Comment by MadmanMartian** [comment]
will be bringing this back as ambience, with a later version.

**Pull request closed without merging**